action-allowedlist/action_allowedlist/actions_parser.py

```diff
@@ -0,0 +1,116 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+import json
+import yaml
+from pathlib import Path
+
+
+def load_json_file(filepath):
+    with open(filepath, 'r') as file:
+        content = file.read()
+        return json.loads(content)
+
+def load_yaml_file(filepath):
```

Comment:
Our Python style guide is to have two blank lines between static functions like this. Please review: https://github.com/Ed-Fi-Alliance-OSS/DevSecOps/tree/main/python. You can skip mypy and mypin.ini for this project, just installing Flake8 and Black.

Once those are installed, you can use VS Code's "Format Document" command and it will integrate with [Black](https://black.readthedocs.io/en/stable/index.html) to reformat a file. Alternately, you can run both tools at the command line:

```
poetry run black action_allowedlist
poetry run flake8 action_allowedlist
```

We also have these tools in the `DevSecOps` repository, and I see that I need to work on the Flake8 configuration there, and I have missed some problems in my own files. Oops.